### DISCUSSION TITLE
Project correction

### DISCUSSION THREAD

**Discussion by gonzalo-munillag** [OpenMined/courses#193] [Project: Privacy Product Specification] [answered]
Hi!

I am not in a hurry whatsoever, but I might have submitted the project wrong, and if that is the case, I might have been waiting for a ghost reviewer all this time hahah
Do you guys have an ETA?

I wish you the best with the corrections, it has to be overwhelming.

Cheers!

**Comment by yemikifouly** [comment, marked as answer]
Hello Gonzalo! 😃 You are completely right to ask. We should have been better at communicating the ETA. So, projects reviews haven't started yet because we are finalizing the training of our mentors. We expect the reviews to start by the end of this week. So, you should be getting the feedback on your submission this weekend or early next week.

Again, sorry for not being clearer about this. Moving forward the ETA for the project will be much, much shorter, hopefully a matter of hours instead of days/weeks. 😬 It is just that the initial training of mentors is taking us longer than expected so for this first course the ETA isn't very good.

Also, I am super glad to hear that you completed the course and submitted the project! 🙂 Woot, woot!!! What did you think of the course?

**Reply by gonzalo-munillag** [reply]
I liked it, the best were the interviews from experts, very very good.
I summarized every lesson and included a highlights document too here: https://github.com/gonzalo-munillag/Private_AI_OpenMined
In case you could use it for something.
I am very eager to start the programming sections of the course.
All the best and a thousand thank yous!

**Comment by adtygan** [comment]
Hahaha! I was thinking of the same thing. I too submitted my project yesterday and am eager for the response. Good luck on finishing the course @gonzalo-munillag !!
